Automattic/jetpack
# Modules: Add Post List support to Jetpack

#### Changes proposed in this Pull Request:

This adds support for the Post List package on Jetpack self hosted sites using the plugin. Previously this feature was only available on WordPress.com.*

#### Other information:


#### Jetpack product discussion


Internal reference: p1658865357288419-slack-C032DVAHMK3

This seems to be the only package that wpcom relies on that Jetpack doesn't, so adding this package to Jetpack directly helps us cleanup wpcom/Jetpack parity. 

Don't merge this until we make a product decision to move forward with it! 

#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






* Spin up a new site and go to yoursite/wp-admin/admin.php?page=jetpack_modules
* Enable the Post List module (it should be off by default)
* Go to your posts and pages list and see that the wp-admin view matches that of wpcom's (easy way to check is by adding a featured image to a post and making sure it shows up there)
* Deactivate the module and make sure it stays off.
